### PR TITLE
task 6 complete

### PR DESCRIPTION
**Pull request by FameTii**

### CODE REVIEW

File: API/app.test.js [hunk 1]
```diff
@@ -108,5 +108,58 @@ describe('GET /api/reviews/:review_id', () => {
...
 });
     
+describe('Get GET /api/reviews/:review_id/comments', () => {
+    it('responds with 200 and returns an array of comments for the given review_id', () => {
+        const review_id = 3
+        return request(app)
+            .get(`/api/reviews/${review_id}/comments`)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toEqual(
```
Comment (Augs0) [inline API/app.test.js:119]: We should also check the length of the array here

File: API/app.test.js [hunk 2]
```diff
@@ -108,5 +108,58 @@ describe('GET /api/reviews/:review_id', () => {
...
+                        body: 'My dog loved this game too!',
+                        review_id: 3,
+                        author: 'mallionaire',
+                        votes: 13,
+                        created_at: '2021-01-18T10:09:05.410Z'
+                      }]
+                )
+                expect(body.comments).toBeSortedBy('created_at', {descending: true});
+            })
+    })
```
Comment (Augs0) [inline API/app.test.js:147]: We should have another 200 test here that returns an empty array if the article exists but has no comments

File: API/app.test.js [hunk 3]
```diff
@@ -108,5 +108,58 @@ describe('GET /api/reviews/:review_id', () => {
...
+    it("should respond with 404 and comment not found", () => {
```
Comment (Augs0) [inline API/app.test.js:157]: This should check if the article exists rather than the comments

File: API/models.js [hunk 4]
```diff
@@ -40,10 +40,32 @@ exports.fetchReviewsWithId = (review_id) => {
...
+                status: 404,
+                msg: `no comments found`
+            })
+        } else {
+            return comments
+        }
+    })
+}
+
+// SELECT * FROM comments WHERE review_id = 3 ORDER BY created_at DESC;
```
Comment (Augs0) [inline API/models.js:67]: Don't forget to remove console logs!

**Review by Augs0 — CHANGES_REQUESTED** [review]
Wonderful job, Fame, really well done. Just a few things to add in the tests!

**Review by pippaamy — APPROVED** [review]
Nice go ahead and merge!
